tests/components/unifiprotect/conftest.py

```diff
@@ -99,28 +55,51 @@ class MockEntityFixture:
     api: Mock
 
 
-MOCK_NVR_DATA = MockNvrData(
-    version=MIN_REQUIRED_PROTECT_V, mac=MAC_ADDR, name="UnifiProtect", id="test_id"
-)
-MOCK_OLD_NVR_DATA = MockNvrData(
-    version=Version("1.19.0"), mac=MAC_ADDR, name="UnifiProtect", id="test_id"
-)
+@pytest.fixture(name="mock_nvr")
+def mock_nvr_fixture():
+    """Mock UniFi Protect Camera device."""
+
+    data = json.loads(load_fixture("sample_nvr.json", integration=DOMAIN))
+    nvr = NVR.from_unifi_dict(**data)
+
+    # disable pydantic validation so mocking can happen
+    NVR.__config__.validate_assignment = False
+
+    yield nvr
 
-MOCK_BOOTSTRAP = MockBootstrap(
-    nvr=MOCK_NVR_DATA, cameras={}, lights={}, sensors={}, viewers={}, liveviews={}
-)
+    NVR.__config__.validate_assignment = True
+
+
+@pytest.fixture(name="mock_old_nvr")
+def mock_old_nvr_fixture():
+    """Mock UniFi Protect Camera device."""
+
+    data = json.loads(load_fixture("sample_nvr.json", integration=DOMAIN))
+    data["version"] = "1.19.0"
+    yield NVR.from_unifi_dict(**data)
```

Comment:
We only need to yield from the fixture when we're inside a context manager or we have some teardown code that we want to run after the yield. If that's not the case we can simply use `return`.